peer.go

```diff
@@ -1429,27 +1429,80 @@ func (p *peer) writeMessage(msg lnwire.Message) error {
 func (p *peer) writeHandler() {
 	var exitErr error
 
+	const (
+		minRetryDelay = 5 * time.Second
```

Comment:
I think we can remove this in favor of just using `writeMessageTimeout` here since it's added in the next commit and don't need to rack the value in two places in the same file.

Reply:
they're different values really, just coincidence that they were chosen to be the same value. in my mind it's not clear that they should always assume the same?